Creature.playground/Contents.swift

```diff
@@ -0,0 +1,77 @@
+import UIKit
+
+//Creation of Creature Structure
+
+struct Creature{
+    var name:String
+    var description:String
+    var isGood:Bool
+    var magicPower:Int
+    
+    //Initialisation of structure
+    
+    init(name: String, description: String, isGood: Bool, magicPower: Int) {
+        self.name = name
+        self.description = description
+        self.isGood = isGood
+        self.magicPower = magicPower
+    }
+    
+    // Computed Property
+    var ability:String
```

Comment:
Another style issue: There should be a space between the colon and String. The open brace should be on the same line as the variable declaration, with a space between String and the opening brace. There should also be a space between the the opening brace and get.

```
var ability: String {
        get {
            return "\(self.name) special ability is \(fibonacciAbility(n: magicPower))"
        }
    }
```